lib/src/mockserver/server.ts

```diff
@@ -66,6 +72,10 @@ export function runMockServer(
   };
 }
 
+function stripDraft(endpointName: string): string {
+  return endpointName.replace("/_draft", "");
```

Comment:
Nit to remove potential unwanted changes such as `/_draft_cats` being affected:
```suggestion
  return endpointName.replace("/_draft/", "/");
```

Reply:
good point. Thanks.